drivers/crypto/Kconfig.nrf52ecb

```diff
@@ -0,0 +1,21 @@
+# NRF52 AES ECB configuration options
+
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+
+menuconfig CRYPTO_NRF52_ECB
```

Comment:
Is it nF52 only? nRF51 has ECB peripheral as well.

Reply:
And nRF53, too.

Reply:
dependency on nRF52 removed in file names, code and in Kconfig. Now it depends on `HAS_HW_NRF_ECB` in Kconfig